src/components/GamesContainer/GameCard/GameCardImage/GameCardImage.jsx

```diff
@@ -0,0 +1,14 @@
+import './GameCardImage.css';
+import notLiked from './notLiked.png';
+
+export default function GameCardImage(props) {
```

Comment:
here you can destructure again: 

function GameCardImage({image})   

so you can use it directly